src/app/page.tsx

```diff
@@ -1,3 +1,9 @@
+import { Header } from "@/components/header-footer";
+
 export default function Page() {
-    return <h1>Hello, Next.js!</h1>
-  }
\ No newline at end of file
+  return (
+    <>
+      <Header></Header>
```

Comment:
a trick with HTML is that if a component has no children like this, you can turn it into a [self-closing tag](https://docureacten.github.io/Component/5-2-How%20to%20Use%20Self-Closing%20Tags) 

```suggestion
      <Header />
```